modules/lm-coursier/src/main/contraband/module.json

```diff
@@ -20,6 +20,46 @@
           "type": "Map[String, String]"
         }
       ]
+    },
+
+    {
+      "name": "ModuleMatcher",
```

Comment:
I'm not sure we need a `ModuleMatcher` under `lmcoursier.definitions`. Just using `lmcoursier.definitions.Module` where it's used should be enough. The less things under `lmcoursier.definitions`, the better.

Reply:
ok

Reply:
Addressed in d74df89